fastify.js

```diff
@@ -338,9 +338,14 @@ function build (options) {
       return instancePrefix
     }
 
-    if (pluginPrefix[0] !== '/') {
+    if (instancePrefix.endsWith('/')) {
```

Comment:
http://node.green/#ES2015-built-ins-well-known-symbols-Symbol-match--String-prototype-endsWith

Unfortunately, we support down to Node v4

Reply:
Ahum... CI is green...
In any case it can be changed with `path.lastIndexOf('/') === (path.length -1)` :)

Reply:
Oh, I see. v4 just doesn't support a regex as the first parameter.